### DISCUSSION TITLE
Update der ROIs wird nicht gespeichert

### DISCUSSION THREAD

**Discussion by griessbx** [jomjol/AI-on-the-edge-device#2512] [Q&A]
Hallo, ich versuche die digitalen ROIs zu verändern, kann diese auch über den Dialog wie gewünscht verändern. Wenn ich die Änderungen dann speichere, kommt der Dialog das die Config.ini modifiziert wurde. Nach dem Reboot sind die alten Werte wieder da, da die config.ini nicht geändert wurde. Irgend eine Idee woran das liegen könnte 
Gruß Roland

**Comment by caco3** [comment]
Bitte Debug log zeigen. Vermutlich hat er ein problem mit dem Schreiben auf die SD-Karte.
